### PR TITLE
Fix GAS initialize

### PR DESCRIPTION
**Pull request by Tommo-L**
Since we already have minted more than 400, 000 GAS in Neo2.x, we need more GAS for migration.

### CODE REVIEW

File: src/neo/SmartContract/Native/Tokens/GasToken.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ internal GasToken()
         internal override void Initialize(ApplicationEngine engine)
         {
             UInt160 account = Blockchain.GetConsensusAddress(Blockchain.StandbyValidators);
-            Mint(engine, account, 30_000_000 * Factor);
+            Mint(engine, account, 100_000_000 * Factor); // Mint for Neo2.x migration. If there is any surplus, we'll burn it later
```
Comment (shargon) [inline src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: 100m is the total supply

Comment (Tommo-L) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: 100m is the GAS total supply of Neo2.x. But for Neo3, it's not fixed.

**Comment by i359** [comment]
It is suggested to keep 3000 0000 unchanged,

The excess can destroy gas in aqvh2pg732yvtnaxegkquei3ya4cvo7d2i，ok？

**Comment by Tommo-L** [comment]
aqvh2pg732yvtnaxegkquei3ya4cvo7d2i ?

**Comment by i359** [comment]
> aqvh2pg732yvtnaxegkquei3ya4cvo7d2i ?

Genesis address

image

File: src/neo/SmartContract/Native/Tokens/GasToken.cs (hunk 1 above)
Comment (erikzhang) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: 50m?

Comment (Tommo-L) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: Still not enough, the total supply of neo2.x is about 43M, and we also need to consider one or two years for neo2.x, which may produce about 18M GAS.

Comment (shargon) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: >  we also need to consider one or two years for neo2.x,

During this years, neo3 will earn gas too

Comment (shargon) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: >  we also need to consider one or two years for neo2.x,

During this years, neo3 will earn gas too

Comment (Tommo-L) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: We should ensure that the users on Neo2.x will not lost their GAS on Neo3

Comment (shargon) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: > We should ensure that the users on Neo2.x will not lost their GAS on Neo3

We can ensure this with less than 100m

Comment (Tommo-L) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs:22]: 70M ?

Comment (shargon) [reply src/neo/SmartContract/Native/Tokens/GasToken.cs]: I think that we can set this value before release neo 3. As @erikzhang said, with 50m would be enough.

**Pull request closed without merging**